notebooks/COS/Requirements.txt

```diff
@@ -0,0 +1,20 @@
+## Requirements
+python3.6
```

Comment:
```suggestion
# requires python3.6 or higher
```
I'm not 100% sure about this, but I think this will break e.g. pip - the python version itself isn't generally managed by pip.  It's a good point this means we don't have an obvious way to state the minimum python version, though. So I'm suggesting here you change it to a comment so that the infromation isn't lost without breaking automated tools.